safe/definitions.py

```diff
@@ -47,6 +47,8 @@
         'property damage, loss of livelihoods and services, social and '
         'economic disruption, or environmental damage. For example; flood, '
         'earthquake, tsunami and volcano are all examples of hazards.')
+        # source UNISDR (2009) Terminology on disaster risk reduction.
```

Comment:
I think it is better to do these as dictionary key / value pairs. If they are comments we cannot use these in reports etc. My suggestion is to follow the style of the impact function metadata (which allows 0 or more citations. Take a look at this example:

```
            'citations': [
                tr('Sengara, W., Suarjana, M., Yulman, M.A., Ghasemi, H., and '
                   'Ryu, H. (2015). '
                   'An empirical fatality model for Indonesia based on '
                   'a Bayesian approach. Submitted for Journal of the '
                   'Geological Society')  # FIXME
            ],
``` 

(taken from ITB impact function' metadata.